.github/workflows/issue-bot.yaml

```diff
@@ -16,13 +16,13 @@ jobs:
 
       - name: Update issue status when assigned
         if: github.event.action == 'assigned'
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --add-label "status:in-progress"
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:accepted" --add-label "status:in-progress"
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Update issue status when unassigned
         if: github.event.action == 'unassigned'
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:in-progress"
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:in-progress" --add-label "status:accepted"
```

Comment:
I was hesitant to do this because of roadmap issues. Maybe the amount of those that we'd unassign are rare enough that it doesn't matter. What if we unassign someone because we've just changed the status to deferred or denied? hm.

Reply:
Ah I see, I just wonder if it gets lost in the shuffle without a `status` label after being unassigned.

Reply:
I was thinking it'd probably be okay if it had both `status:accepted` and `status:in-progress` at the same time.

Reply:
Got it, so `status:accepted` is never removed until completed, basically.